src/templates/community-champion-profile.js

```diff
@@ -241,9 +242,24 @@ const CommunityChampionProfile = props => {
     const imageUrl = champion.image ? champion.image.url : null;
     const useMobileLayout = useMedia(screenSize.upToSmallDesktop);
     const metadata = useSiteMetadata();
+    const fullUrl = removePathPrefixFromUrl(`${metadata.siteUrl}${slug}`);
+    const seoTitle = `${fullName} - ${metadata.title}`;
```

Comment:
```suggestion
    const seoTitle = `${fullName} - MongoDB Community Champion`;
```

Nitpick but this feels better to me

Reply:
done! :)